robot_exploration:

  world_frame: "map"
  base_frame: "pelvis"

  min_nav_target_distance: 0.15  # [m]

  distance_target_object: 0.85 # [m]

  exploration:
    timer_frontiers: 1.5 # [Seconds]
    min_dist_frontier_robot: 0.25 # [m]

    cost_function:
      n_points: 5.0
      euclidean_distance: 20.0 # Closer to the robot, the better (multiplied by the inverse of the distance)
      rotation_distance: 45.0
      distance_prev_target: 35.0 # Close to previous target is better, to avoid switching from one side of the map to the other one
      close_frontiers: 25.0